packages/gui/src/electron/main.tsx

```diff
@@ -8,16 +8,21 @@ import {
   IncomingMessage,
   Menu,
   nativeImage,
+  protocol,
 } from 'electron';
 import { initialize } from '@electron/remote/main';
 import path from 'path';
-import React from 'react';
+import React, { isValidElement } from 'react';
```

Comment:
`isValidElement` isn't used

```suggestion
import React from 'react';
```